Model/Laybuy.php

```diff
@@ -545,6 +550,43 @@ protected function shouldBeInvoiced(\Magento\Sales\Model\Order $order)
 
         return true;
     }
+
+    /**
+     * @param \Magento\Payment\Model\InfoInterface $payment
+     * @param float $amount
+     * @return $this
+     */
+    public function refund(\Magento\Payment\Model\InfoInterface $payment, $amount)
+    {
+
+        // Laybuy module stores remote order reference as <orderId>_<token>, so we need to split it out for the refund request.
+        $laybuyOrderId = $payment->getAdditionalInformation(Config::LAYBUY_FIELD_REFERENCE_ORDER_ID);
+
+        if (!$laybuyOrderId) {
+            $this->logger->debug(['Unable to process refund, payment details are missing: ' . $payment->getId()]);
```

Comment:
Might need to be `=>` instead of `.` maybe (based on the other logger->debug calls)?

Reply:
Shouldn't really matter, with this it would just print as:
`0: Unable to process refund, payment details are missing: 123456`

But can update for consistency

Reply:
And done

Reply:
:star: